scripts/azureml-assets/azureml/assets/copy_assets.py

```diff
@@ -31,9 +35,33 @@ def copy_asset(asset_config: assets.AssetConfig,
         # Skip a released version
         return None
 
+    if check_previous_release and asset_config.type == AssetType.ENVIRONMENT:
+        # Skip if previous version was not released
+        previous_release_version = assets.get_latest_release_tag_version(asset_config, release_directory_root)
+
+        if previous_release_version is not None:
+            image = asset_config.extra_config_as_object().get_full_image_name()
+
+            # Check against MCR by making a manifest call to see if tag exists
+            (hostname, repo) = image.split("/", 1)
+
+            try:
+                response = assets.get_manifest(previous_release_version, hostname, repo)

```

Comment:
We don't care about the result, just that we didn't get an error.
```suggestion
                _ = assets.get_manifest(previous_release_version, hostname, repo)
```